integration-tests/src/test/resources/oke/terraform/twoclusters/cluster.tf

```diff
@@ -2,104 +2,156 @@
+    service_lb_subnet_ids = [oci_core_subnet.oke-subnet-loadbalancer-1.id, oci_core_subnet.oke-subnet-loadbalancer-2.id]
 
     #Optional
     add_ons {
       #Optional
-      is_kubernetes_dashboard_enabled = "${var.cluster_options_add_ons_is_kubernetes_dashboard_enabled}"
-      is_tiller_enabled = "${var.cluster_options_add_ons_is_tiller_enabled}"
+      is_kubernetes_dashboard_enabled = var.cluster_options_add_ons_is_kubernetes_dashboard_enabled
+      is_tiller_enabled               = var.cluster_options_add_ons_is_tiller_enabled
     }
   }
 }
 
 resource "oci_containerengine_node_pool" "tfsample_node_pool1" {
-	#Required
-	cluster_id = "${oci_containerengine_cluster.tfsample_cluster1.id}"
-	compartment_id = "${var.compartment_ocid}"
-	kubernetes_version = "${var.node_pool_kubernetes_version}"
-	name = "${var.node_pool_name1}"
-	node_image_name = "${var.node_pool_node_image_name}"
-	node_shape = "${var.node_pool_node_shape}"
-        #subnet_ids = ["${oci_core_subnet.oke-subnet-worker-1.id}", "${oci_core_subnet.oke-subnet-worker-2.id}","${oci_core_subnet.oke-subnet-worker-3.id}"]
-        subnet_ids = ["${oci_core_subnet.oke-subnet-worker-1.id}", "${oci_core_subnet.oke-subnet-worker-2.id}"]
+  #Required
+  cluster_id         = oci_containerengine_cluster.tfsample_cluster1.id
+  compartment_id     = var.compartment_ocid
+  kubernetes_version = var.node_pool_kubernetes_version
+  name               = var.node_pool_name1
+  node_image_name    = var.node_pool_node_image_name
+  node_shape         = var.node_pool_node_shape
+
+  #subnet_ids = ["${oci_core_subnet.oke-subnet-worker-1.id}", "${oci_core_subnet.oke-subnet-worker-2.id}","${oci_core_subnet.oke-subnet-worker-3.id}"]
+  subnet_ids = [oci_core_subnet.oke-subnet-worker-1.id, oci_core_subnet.oke-subnet-worker-2.id]
 
-	#Optional
-	quantity_per_subnet = "${var.node_pool_quantity_per_subnet}"
-	ssh_public_key = "${var.node_pool_ssh_public_key}"
+  #Optional
+  quantity_per_subnet = var.node_pool_quantity_per_subnet
+  ssh_public_key      = var.node_pool_ssh_public_key
 }
 
 resource "oci_containerengine_node_pool" "tfsample_node_pool2" {
   #Required
-  cluster_id = "${oci_containerengine_cluster.tfsample_cluster2.id}"
-  compartment_id = "${var.compartment_ocid}"
-  kubernetes_version = "${var.node_pool_kubernetes_version}"
-  name = "${var.node_pool_name2}"
-  node_image_name = "${var.node_pool_node_image_name}"
-  node_shape = "${var.node_pool_node_shape}"
+  cluster_id         = oci_containerengine_cluster.tfsample_cluster2.id
+  compartment_id     = var.compartment_ocid
+  kubernetes_version = var.node_pool_kubernetes_version
+  name               = var.node_pool_name2
+  node_image_name    = var.node_pool_node_image_name
+  node_shape         = var.node_pool_node_shape
+
   #subnet_ids = ["${oci_core_subnet.oke-subnet-worker-1.id}", "${oci_core_subnet.oke-subnet-worker-2.id}","${oci_core_subnet.oke-subnet-worker-3.id}"]
```

Comment:
delete commented out lines

Reply:
I left them intentionally , so we can switch between2 and 3 nodes in cluster